lib/remote_helper.py

```diff
@@ -694,14 +694,9 @@ def start_remote_appcontroller(cls, host, keyname, is_verbose):
     cls.scp(host, keyname, cls.MONIT_APPCONTROLLER_CONFIG_FILE,
       '/etc/monit/conf.d/appscale-controller-17443.cfg', is_verbose)
 
-    # Start up monit.
-    cls.ssh(host, keyname, 'monit quit; ', is_verbose)
+    # Re-start up monit to make sure it is running and it did reloaded its
+    # configuration (and consequently starts the controller).
     cls.ssh(host, keyname, 'service monit start', is_verbose)
-    time.sleep(1)
-
-    # Start the AppController.
-    cls.ssh(host, keyname, 'monit start -g controller', is_verbose)
```

Comment:
It shouldn't be running at this point, that's why we copied over the monit configuration. The restart of monit will start the controller.